src/main/scopes/js/import-matchers/js-named-import-matcher.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type { JsFunction, JsImport, JsImportMatcher, JsToken } from '../interfaces.js'
+
+/**
+ * Identifies JsTokens and JsFunctions that have been imported as named imports,
+ * and returns an import element match (if any) or undefined otherwise.
+ */
```

Comment:
Same thing here. You can consolidate these docs on the `findMatch` method and keep the class level docs pretty basic.